frontend/ui/elements/screensaver_menu.lua

```diff
@@ -177,6 +178,38 @@ return {
                 end,
                 separator = true,
             },
+            {
+                text = _("Screensaver message position"),
+                sub_item_table = {
+                    {
+                        text = _("Top"),
+                        checked_func = function()
+                            return messagePosition() == "top" or messagePosition() == nil
+                        end,
+                        callback = function()
+                            G_reader_settings:saveSetting("screensaver_message_position", "top")
+                        end
+                    },
+                    {
+                        text = _("Middle"),
+                        checked_func = function()
+                            return messagePosition() == "middle" or messagePosition() == nil
```

Comment:
I guess you want the `or messagePosition() == nil` here, but not above and below.

Reply:
whoops, I blame Homer Simpson for that one!